src/TiffImages.jl

```diff
@@ -37,12 +39,25 @@ export memmap
 
 ## Precompilation helper
 mktemp() do fpath, _
-    for t in [N0f8, N0f16, Float32, Float64]
-        for c in [Gray, GrayA, RGB, RGBA]
-            TiffImages.save(fpath, rand(c{t}, 2, 2))
+    for t in Any[N0f8, N0f16, Float32, Float64]
+        for c in Any[Gray, GrayA, RGB, RGBA], sz in ((2, 2), (2, 2, 2))
+            TiffImages.save(fpath, rand(c{t}, sz))
```

Comment:
This works locally

```suggestion
            TiffImages.save(fpath, rand(c{t}, sz))
            Sys.iswindows() && GC.gc()
```

Reply:
Huh, weird. Do you understand why? It's the loading via `mmap` that's presumably causing the problem, so it's not clear to me why GCing after the save helps. Maybe it's just an extra GC on each loop that does a more thorough cleaning?